File: nodejs/wordcount/interactors.js
const wordcount = require("./wordcount.js");
const stopwordsprovider = require("./stopwordsprovider.js");

exports.Start = function(text) {
    let words = wordcount.SplitTextIntoWords(text);
    let stopwords = stopwordsprovider.ReadStopwords();
    let filteredWords = wordcount.FilterStopwords(words, stopwords);
    let numberOfWords = wordcount.CountWords(filteredWords);
    return numberOfWords;
};

File: python/wordcount/interactors.py
from stopwords_provider import read_stopwords
from wordcount import split_into_words, filter_stopwords


def start(text):
    words = split_into_words(text)
    stopwords = read_stopwords()
    filtered_words = filter_stopwords(words, stopwords)
    return len(filtered_words)
